yellowbrick/features/projection.py

```diff
@@ -21,6 +21,7 @@
 import matplotlib as mpl
 import matplotlib.pyplot as plt
 from mpl_toolkits.axes_grid1 import make_axes_locatable
+import mpl_toolkits.mplot3d  # noqa
```

Comment:
I know that this was in the PCA file - was there an error when this import wasn't here and we used 3D axes? It seemed like Manifold was working fine.

Reply:
Yup, It gave an error when this was not included. I had a bit of research on this and this is what I found.
1. If I remove this import from `projection.py`, it now gives an error for `manifold` too.
2. Then I tried adding the import to PCA and now the tests for manifold passes.